docs/tensorboard_in_notebooks.ipynb

```diff
@@ -517,4 +513,4 @@
       ]
     }
   ]
-}
```

Comment:
Revert EOL change—IPython notebooks are blobs without trailing NL.
Likewise in other notebooks.

Reply:
Done